x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/persistence/DataFrameTransformsConfigManager.java

```diff
@@ -51,7 +52,7 @@
     public static final Map<String, String> TO_XCONTENT_PARAMS;
     static {
         Map<String, String> modifiable = new HashMap<>();
-        modifiable.put("for_internal_storage", "true");
+        modifiable.put(DataFrameField.FOR_INTERNAL_STORAGE, "true");
         TO_XCONTENT_PARAMS = Collections.unmodifiableMap(modifiable);
     }
```

Comment:
If you use `Collections.singletonMap` then you can replace lines 52-57 with 1 line.  But obviously what you have now is extensible if there is ever a need to add another parameter.  I don't really mind if you change it.